mmv1/products/networkservices/MulticastDomainActivation.yaml

```diff
@@ -111,36 +142,41 @@ properties:
         description: |-
           Aggregated egress Packet-Per-Second for all multicast groups in the domain
           in this zone.
+        default_from_api: true
       - name: aggrIngressPps
         type: String
         description: |-
           Aggregated ingress Packet-Per-Second for all multicast groups in the domain
           in this zone. Default to (aggregated_egress_pps /
           max_per_group_subscribers) * 2.
+        default_from_api: true
       - name: avgPacketSize
-        type: Integer
+        type: String
```

Comment:
@jgforsberg I believe the failure is due to this change, where presumably there is a config value in a test still using a regular integer.